### PR TITLE
Update GET-Properties-Reference.md

### LINKED ISSUE #4: Get Table Number is missing an example
Please add the example code
remember to use the triple tick marks
ie 
"```" which renders as 
```

### PR DESCRIPTION
**Pull request by Aaron-Gor**
@lsnovich Fixed #4

### CODE REVIEW

File: GET-Properties-Reference.md [hunk 1]
```diff
@@ -18,6 +19,7 @@ This section shows the properties, data type, and descriptions required for gene
 Possible server status codes following GET action (Get HTTP Status Codes) can be found [here](https://github.com/Aaron-Gor/API-Docs/blob/main/README.md)
 
 
+
 ## EXAMPLE JSON CODING FOR GETTING THE BILL
```
Comment (lsnovich) [inline GET-Properties-Reference.md:23]:
```suggestion
## JSON Response Schema
```

File: GET-Properties-Reference.md [hunk 2]
```diff
@@ -18,6 +19,7 @@ This section shows the properties, data type, and descriptions required for gene
...
 
 }
```
Comment (lsnovich) [inline GET-Properties-Reference.md:25]:
```suggestion
```JSON
}
```

Comment (lsnovich) [reply GET-Properties-Reference.md:25]: you also need 
the tick marks (```) at the end

**Review by lsnovich — CHANGES_REQUESTED** [review]

**Comment by Aaron-Gor** [comment]
@lsnovich PR #11 closed as requested.

**Pull request closed without merging**